CONVENTIONS.md

```diff
@@ -0,0 +1,80 @@
+## Git Commit Guidelines
+
+These rules are adopted from [the AngularJS commit conventions](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/).
+
+### Commit Message Format
+
+Each commit message starts with a **type**, a **scope**, and a **subject**.
+
+Below that, the commit message has a **body**.
+
+- **type**: what type of change this commit contains.
+- **scope**: what item of code this commit is changing.
+- **subject**: a short description of the changes.
+- **body** (optional): a more in-depth description of the changes
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+```
+
+Examples:
+```
+feat(ruler): add inches as well as centimeters
+```
+```
+fix(protractor): fix 90 degrees counting as 91 degrees
+```
+```
+refactor(pencil): use graphite instead of lead
+
+Closes #640.
+
+Graphite is a much more available resource than lead, so we use it to lower the price.
+```
+```
+fix(pen): use blue ink instead of red ink
+
+BREAKING CHANGE: Pen now uses blue ink instead of red.
+
+To migrate, change your code from the following:
+
+`pen.draw('blue')`
+
+To:
+
+`pen.draw('red')`
+```
+
+Any line of the commit message should not be longer 100 characters. This allows the message to be easier
+to read on github as well as in various git tools.
+
+### Type
+Is recommended to be one of the below items. Only **feat** and **fix** show up in the changelog, in addition to breaking changes (see breaking changes section at bottom).
+
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug or adds a feature
+* **test**: Adding missing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
+  generation
```

Comment:
No `build` and no `ci`? 🤔 

And do we really want to copy the commit conventions in this file? Wouldn't it be enough to link to https://www.conventionalcommits.org/?

Reply:
I think it is easier to read like this, Normally people do not like clicking several links to get information and it would save them time like this - and also it saves us time later, as they would stick to the conventions. But what I can do is adding the link additionally, if anyone wants to read more about it.

> No build and no ci? 🤔

Never used them, and never saw someone using them (hm?). Do you want them? :D

Reply:
> I think it is easier to read like this, Normally people do not like clicking several links to get information and it would save them time like this - and also it saves us time later, as they would stick to the conventions. But what I can do is adding the link additionally, if anyone wants to read more about it.

Ok.

>> No build and no ci? 🤔
>
> Never used them, and never saw someone using them (hm?). Do you want them? :D

I discovered them recently and I like them somehow. 😊
Another additional type would be `perf` for performance.